ipynb/android/Android_Workloads.ipynb

```diff
@@ -570,7 +570,9 @@
     "        results[conf_name][wload_name] = res\n",
     "\n",
     "    # Save collected results\n",
-    "    res_file = os.path.join(te.res_dir, conf_name, 'results.json')\n",
+    "    conf_dir = os.path.join(te.res_dir, conf_name)\n",
+    "    if not os.path.isdir(conf_dir): os.mkdir(conf_dir)\n",
```

Comment:
I've just discovered that loops have else in python, which is awesome (I haven't used python much before :">).